tests/integration/functional/cli/test_model.py

```diff
@@ -240,6 +240,7 @@ def test_update_model(
 
     # Check the result
     assert result.exit_code == 0
+    assert "updated successfully." in result.output
```

Comment:
I meant more to test the actual model itself (get it via the `Client()` etc), and not just the output.